doc/github_handler.rst

```diff
@@ -0,0 +1,60 @@
+GitHub Handler
+==============
+
+GitHub Handler is a Universum mode that serves as GitHub Application, helping to perform and report checks on
+new commits to a repository. In this mode Universum can create new check runs on GitHub and trigger an already
+set up automation server to perform these checks. GitHub Handler parses all required params and passes them
+to the triggered builds.
+
+
+How to set up GitHub auto check using Unversum
+----------------------------------------------
+
+Universum in default, 'main' mode when set up to ``--vcs-type=github`` and with ``--report-to-review`` passed
+as parameter, can post `check run` statuses to GitHub to be depicted on 'Checks' page in pull requests or as
+a simple icon near any checked commit. To do so, it needs `a set of parameters <args.html#GitHub>`__
+to be passed to it.
+
+.. note::
+
+    During these checks, if some steps were using :doc:`Universum analyzers <code_report>`,
+    and any issues are found, they will also be reported in comments.
+
+These required parameters are sent by GitHub via web-hooks after the reuired `check run` is created. To do so,
+a GitHub Application is required, and GitHub Handler could be used as such.
+
+.. note::
+
+    GitHub Handler is a simple script, receiving web-hook payload and event as parameters.
+    To work as GitHub Application it still needs a server to receive actual web-hooks and pass their
+    contents as parameters to Handler.
+
+One of the easiest ways to make it work is to put the GitHub Handler execution into an automation server
+(such as Jenkins) job triggered by incoming web-hooks from GitHub. GitHub Handler parses the incoming
+web-hook event (passed via "x-github-event" header) and payload content. If a new `check run` is required,
+it sends a request to GitHub to create one. If the `check run` is already created, it parses the required parameters
+and triggers the set up automation server to run a previously described check by usual Universum.
```

Comment:
In my opinion, the reference to "usual Universum" doesn't help here.
It must be at least mentioned in advance.